<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.WcD22Zf' shared service.

return $this->privates['.service_locator.WcD22Zf'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('accountRepo' => function (): \App\Repository\CeAccountRepository {
    return ($this->privates['App\Repository\CeAccountRepository'] ?? $this->load('getCeAccountRepositoryService.php'));
}, 'chargeRep' => function (): \App\Repository\CeChargeRepository {
    return ($this->privates['App\Repository\CeChargeRepository'] ?? $this->load('getCeChargeRepositoryService.php'));
}, 'order' => function (): \App\Entity\Ceorder {
    return ($this->privates['.errored..service_locator.WcD22Zf.App\Entity\Ceorder'] ?? $this->load('getCeorder3Service.php'));
}));
